Sources/LanguageServerProtocol/Requests/FormattingRequests.swift

```diff
@@ -106,4 +124,12 @@ public struct FormattingOptions: Codable, Hashable {
 
   /// Trim all newlines after the final newline at the end of the file.
   public var trimFinalNewlines: Bool?
+
+  public init(tabSize: Int, insertSpaces: Bool, trimTrailingWhitespace: Bool? = nil, insertFinalNewline: Bool? = nil, trimFinalNewlines: Bool? = nil) {
```

Comment:
This line is more than 120 characters and swift-format will complain about it. Could you run `swift-format` on your changes. What I usually do, is build `swift-format` locally and then run 

```bash
~/src/swift-format/.build/arm64-apple-macosx/release/swift-format lint --strict --recursive Sources/ Tests/ Package.swift
```

(and yes, we should really have documentation/scripts for that.